package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This is an awesome multi-player game.",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npx mocha -c --recursive"
```

Comment:
Another bit of metadata that `package.json` holds is _scripts_: they're like functions that we can call from the _shell_.

I've added a script called `test` which will run the new `mocha` dependency that you'll have installed from the earlier step. Mocha is a module that allows us to write tests to prove that our game works. It'll be useful because it means you can totally try new things in the code and the tests will let you know if something that previously worked has been broken.

To run the tests, type the following at your shell:

```shell
npm t
```

You should see output like this:

<img width="590" alt="Screen Shot 2019-07-30 at 19 07 18" src="https://user-images.githubusercontent.com/1855109/62153849-64da2a00-b2fd-11e9-8022-e60fe438d58c.png">